spec/tasks/whitehall_importer_spec.rb

```diff
@@ -76,6 +86,24 @@
     expect(edition.update_type).to eq("major")
   end
 
+  it "adds users who have never logged into Content Publisher" do
+    importer = Tasks::WhitehallImporter.new(123, import_data)
+    importer.import
+
+    expect(User.last.uid).to eq "36d5154e-d3b7-4e3e-aad8-32a50fc9430e"
+    expect(User.last.name).to eq "A Person"
+    expect(User.last.email).to eq "a-publisher@department.gov.uk"
+    expect(User.last.organisation_slug).to eq "a-government-department"
+    expect(User.last.organisation_content_id).to eq "01892f23-b069-43f5-8404-d082f8dffcb9"
+  end
+
+  it "does not add users who have logged into Content Publisher" do
+    importer = Tasks::WhitehallImporter.new(123, import_data)
+    User.create!(uid: "36d5154e-d3b7-4e3e-aad8-32a50fc9430e")
+
+    expect { importer.import }.not_to(change { User.count })
+  end
```

Comment:
This is great 👍  mergeable. But I wonder if it's worth testing that it doesn't overwrite users who have logged into Content Publisher, in case their name/email has changed? The code doesn't overwrite but may be an idea to enforce that with a test.

Reply:
It won't matter if their data is overwritten, since it'll get updated from Signon next time they login